public/main.js

```diff
@@ -20,7 +25,69 @@ class HashRouter {
   }
 }
 
-const $views = document.querySelectorAll('.view')
 const router = new HashRouter($views)
 
+function searchParser(item) {
+  let searchString = item.value
+
+  let splitString = searchString.split(',')
+  splitString.splice(1, 100)
+  let newString = splitString.toString().split(' ')
+  let sendString = newString.reduce((total, element) => {
+    return total + element
+  })
+  return sendString
+}
+
+function renderTwitterElements(response) {
+  const $tweetList = document.createElement('ul')
+
+  response.forEach(element => {
+    const $tweetLi = document.createElement('li')
+    const $mediaImg = document.createElement('div')
+    const $img = document.createElement('img')
+    const $name = document.createElement('h5')
+    const $text = document.createElement('p')
+    const $mediaBody = document.createElement('div')
+
+    for (const property in element) {
+      if (property === 'text') {
+        $text.textContent =  element[property]
+      }
+      else if (property === 'user') {
+        $name.textContent = element[property].name
+        $img.src = element[property].profile_image_url
+      }
+    }
+
+    $tweetLi.classList.add('media')
+    $name.classList.add('media-heading')
+    $img.classList.add('media-object')
+    $mediaImg.classList.add('media-left')
+    $mediaBody.classList.add('media-body')
+
+    $mediaImg.appendChild($img)
+    $mediaBody.appendChild($name)
+    $mediaBody.appendChild($text)
+    $tweetLi.appendChild($mediaImg)
+    $tweetLi.appendChild($mediaBody)
+    $tweetList.appendChild($tweetLi)
+  })
+  $tweetList.classList.add('media-list')
+  return $tweetList
+}
+
+$mapPage.addEventListener('mouseenter', () => {
+  let fetchItem = searchParser($searchItem)
+  $twitterContainer.innerHTML = ''
+
+  fetch('http://localhost:3000/tweets/' + fetchItem)
```

Comment:
Your `fetch` url can just be relative (i.e. `'/tweets' + fetchItem`). You don't want to hardcode the server into your client code.